styles/designs/carnival/parts/_exercises-components.scss

```diff
@@ -244,7 +244,7 @@ $Exercise__ProblemSolution__Number: (
   _name: "ProblemSolutionNumber",
   _subselector: ' > .os-number',
   _properties: (
-    display: table-cell,
+    display: inline,
```

Comment:
could books other than nursing-internal be impacted by this change as well?

Reply:
Yep, you're right about this. I have reverted the previous change and instead set the `width` and `text-align` properties explicitly in the nursing internal stylesheet.